src/stravalib/model.py

```diff
@@ -738,7 +740,10 @@ class SummarySegment(strava_model.SummarySegment, BoundClientEntity):
     start_latlng: Optional[LatLon] = None
     end_latlng: Optional[LatLon] = None
     athlete_pr_effort: Optional[AthletePrEffort] = None
-    activity_type: Optional[RelaxedActivityType] = None  # type: ignore[assignment]
+    # TODO: mypy error - SummarySegment defines as  literal with two values
+    # activity_type: Optional[Literal["Ride", "Run"]] = None
+    # We could force ignore via cast??
+    activity_type: Optional[RelaxedActivityType] = None
```

Comment:
Mypy is failing here - 

```
src/stravalib/model.py:746: error: Incompatible types in assignment (expression has type "Optional[RelaxedActivityType]", base class "SummarySegment" defined the type as "Optional[Literal['Ride', 'Run']]")  [assignment]
```
i understand why it's failing but i don't understand the best way to address it. Do we need to use cast somewhere? or how do we tell mypy this typing mismatch is in fact ok?